lib/gtk/v3_4/gtk_dialogs.nit

```diff
@@ -23,299 +23,315 @@ in "C Header" `{
 	#include <gtk/gtk.h>
 `}
 
-#Create popup windows
-#@https://developer.gnome.org/gtk3/stable/GtkDialog.html
+# Create popup windows
+# See: https://developer.gnome.org/gtk3/stable/GtkDialog.html
 extern class GtkDialog `{GtkDialog *`}
 	super GtkWindow
 
-	new is extern `{
-		return (GtkDialog *)gtk_dialog_new( );
+	new `{
+		return (GtkDialog *)gtk_dialog_new();
 	`}
 
-	new with_buttons( title : String, parent : GtkWindow, flags : GtkDialogFlags) is extern import String.to_cstring`{
-		return (GtkDialog *)gtk_dialog_new_with_buttons( String_to_cstring( title ), parent, flags, "", NULL );
+	new with_buttons(title: String, parent: GtkWindow, flags: GtkDialogFlags)
+	import String.to_cstring `{
+		return (GtkDialog *)gtk_dialog_new_with_buttons(
+			String_to_cstring(title), parent, flags, "", NULL);
 	`}
 
-	fun run is extern `{
-		gtk_dialog_run( recv );
+	fun run `{
+		gtk_dialog_run(recv);
 	`}
-
 end
 
-#Display information about an application
-#@https://developer.gnome.org/gtk3/stable/GtkAboutDialog.html
+# Display information about an application
+# See: https://developer.gnome.org/gtk3/stable/GtkAboutDialog.html
 extern class GtkAboutDialog `{GtkAboutDialog *`}
 	super GtkDialog
 
-	new is extern `{
-		return (GtkAboutDialog *)gtk_about_dialog_new( );
+	new `{
+		return (GtkAboutDialog *)gtk_about_dialog_new();
 	`}
 
-	fun program_name : String import NativeString.to_s `{
-		return NativeString_to_s( (char *)gtk_about_dialog_get_program_name( recv ) );
+	fun program_name: String import NativeString.to_s `{
+		return NativeString_to_s((char *)gtk_about_dialog_get_program_name(recv));
```

Comment:
Yup, this is a real problem. Note that it was written before NativeString::to_s_with_copy.  It must be fixed... in a future PR.